Formula/vcluster.rb

```diff
@@ -0,0 +1,39 @@
+class Vcluster < Formula
+  desc "Creates fully functional virtual k8s cluster inside host k8s cluster's namespace"
+  homepage "https://www.vcluster.com"
+  url "https://github.com/loft-sh/vcluster.git",
+      tag:      "v0.7.1",
+      revision: "dc0ff6f96e9c96fe2caa77e79c2dffc921b4fd49"
+  license "Apache-2.0"
+  head "https://github.com/loft-sh/vcluster.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "helm"
+  depends_on "kubernetes-cli"
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.commitHash=#{Utils.git_head}
+      -X main.buildDate=#{time.iso8601}
+      -X main.version=#{version}
+    ]
+    system "go", "build", "-mod", "vendor", *std_go_args(ldflags: ldflags), "./cmd/vclusterctl/main.go"
+    (zsh_completion/"_vcluster").write Utils.safe_popen_read(bin/"vcluster", "completion", "zsh")
+    (bash_completion/"vcluster").write Utils.safe_popen_read(bin/"vcluster", "completion", "bash")
+  end
+
+  test do
+    help_output = "vcluster root command"
+    assert_match help_output, shell_output("#{bin}/vcluster --help")
+
+    create_output = <<~EOS
+      there is an error loading your current kube config 
+      (invalid configuration: no configuration has been provided, 
+      try setting KUBERNETES_MASTER environment variable), 
+      please make sure you have access to a kubernetes cluster and the command 
```

Comment:
```suggestion
      there is an error loading your current kube config\s
      (invalid configuration: no configuration has been provided,\s
      try setting KUBERNETES_MASTER environment variable),\s
      please make sure you have access to a kubernetes cluster and the command\s
```
Yep, `brew` doesn't like the trailing spaces so we need `\s`s.

Reply:
and also each next line is considered as \n new line, resulting in a test failure. So I reverted back to as it was before.

Reply:
Do you've any suggestions for this? Attaching error for reference
```
➜  homebrew-core git:(master) ✗ brew test --verbose vcluster                                                          
==> Testing vcluster
==> /opt/homebrew/Cellar/vcluster/0.7.1/bin/vcluster --help
==> /opt/homebrew/Cellar/vcluster/0.7.1/bin/vcluster create vcluster -n vcluster --create-namespace
Error: vcluster: failed
An exception occurred within a child process:
  Minitest::Assertion: Expected /there\ is\ an\ error\ loading\ your\ current\ kube\ config\ \n\(invalid\ configuration:\ no\ configuration\ has\ been\ provided,\ \ntry\ setting\ KUBERNETES_MASTER\ environment\ variable\),\ \nplease\ make\ sure\ you\ have\ access\ to\ a\ kubernetes\ cluster\ and\ the\ command\ \n`kubectl\ get\ namespaces`\ is\ working\ \n/ to match "\e[1;31m[fatal]  \e[0mthere is an error loading your current kube config (invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable), please make sure you have access to a kubernetes cluster and the command `kubectl get namespaces` is working\n".
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/minitest-5.15.0/lib/minitest/assertions.rb:183:in `assert'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/minitest-5.15.0/lib/minitest/assertions.rb:295:in `assert_match'
/opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/vcluster.rb:37:in `block in <class:Vcluster>'
/opt/homebrew/Library/Homebrew/formula.rb:2112:in `block (3 levels) in run_test'
/opt/homebrew/Library/Homebrew/utils.rb:590:in `with_env'
/opt/homebrew/Library/Homebrew/formula.rb:2111:in `block (2 levels) in run_test'
/opt/homebrew/Library/Homebrew/formula.rb:948:in `with_logging'
/opt/homebrew/Library/Homebrew/formula.rb:2110:in `block in run_test'
/opt/homebrew/Library/Homebrew/mktemp.rb:63:in `block in run'
/opt/homebrew/Library/Homebrew/mktemp.rb:63:in `chdir'
/opt/homebrew/Library/Homebrew/mktemp.rb:63:in `run'
/opt/homebrew/Library/Homebrew/formula.rb:2363:in `mktemp'
/opt/homebrew/Library/Homebrew/formula.rb:2104:in `run_test'
/opt/homebrew/Library/Homebrew/test.rb:43:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:93:in `block in timeout'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `block in catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:108:in `timeout'
/opt/homebrew/Library/Homebrew/test.rb:48:in `<main>'
```